lib/mongo/database/view.rb

```diff
@@ -93,29 +93,29 @@ def initialize(database)
 
       private
 
-      def collections_info(server, session, &block)
-        cursor = Cursor.new(self, send_initial_query(server, session), server, session: session)
+      def collections_info(server, session, name_only = false, &block)
```

Comment:
For future extensibility should this be an options hash?

Reply:
Good idea!